docs/code-quality/analyzer-rule-sets.md

```diff
@@ -65,7 +65,7 @@ The process for setting the active rule set is a little different depending on w
 
 #### .NET Core
 
-To make a rule set the active rule set for analysis in .NET Core or .NET Standard projects, manually add the **CodeAnalysisRuleSet** property to your project file. For example, the following code snippet sets `HelloWorld.ruleset` as the active rule set.
+To make a rule, set the active rule set for analysis in .NET Core or .NET Standard projects, manually add the **CodeAnalysisRuleSet** property to your project file. For example, the following code snippet sets `HelloWorld.ruleset` as the active rule set.
```

Comment:
Please revert, and instead use “ruleset”

Reply:
Will fix it. Thank you for reviewing!

Reply:
Hi, again, @mavasani, I noticed there are a few UI elements with "ruleset" named differently:
- **Run this ruleset**
- **CodeAnalysisRuleSet**

Which spelling is correct: ruleset or RuleSet?

Reply:
"rule set" is the current spelling in the docs. Thanks.